app/src/main/kotlin/compiler/ast/Expression.kt

```diff
@@ -113,4 +113,13 @@ sealed class Expression : AstNode {
         val resultWhenFalse: Expression,
         override val location: LocationRange? = null,
     ) : Expression()
+
+    companion object {
+        fun getValueOfLiteral(expr: Expression) = when (expr) {
+            is UnitLiteral -> 0L
```

Comment:
We have a constant for that (in IFTNode.kt).

Reply:
Actually, unit values should not even exist in the generated code, so whichever representation is chosen does not really matter.

Reply:
I agree with that, it just looks little inconsistent if we have a constant UNIT_VALUE and use it in some places while in the others we have literal `0L`. So maybe it would be better idea to remove the constant (since it has no semantic meaning).